src/Avalonia.Diagnostics/Views/ControlDetailsView.cs

```diff
@@ -30,12 +31,14 @@ public ControlDetailsViewModel ViewModel
             private set
             {
                 SetValue(ViewModelProperty, value);
-                _grid[GridRepeater.ItemsProperty] = value?.Properties;
+                //_grid[GridRepeater.ItemsProperty] = value?.Properties;
             }
         }
 
         private void InitializeComponent()
         {
+            AvaloniaXamlLoader.Load(this);
+            return;
             Func<object, IEnumerable<Control>> pt = PropertyTemplate;
```

Comment:
This code will never get executed. Is this debug code?

Reply:
nope, i just haven't cleaned up the code (old code to be able to check it out quickly in old mode), but if this pr is liked and going to be merged i can clean up some old unused code

Reply:
Ah ok. I'm actually working on something similar over at https://github.com/grokys/devtools but I'm not sure how to include it in the main repo as it has a dependency on DataGrid.

If you want to clean this code up though, this might be a good interim solution.